eks/verify_kubera_on_eks.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+ 
+set +e
+
+## Install helm
+curl -fsSL -o get_helm.sh \
+https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3
+
+chmod 700 get_helm.sh
+./get_helm.sh
+helm version
+
+## Creates secret this secret will help us to pull images from registry
+kubectl create secret docker-registry directoronprem-registry-secret \
```

Comment:
are these two secrets or a single secret.
add echo to log

Reply:
Single secret